pkg/plugins/plugin.go

```diff
@@ -49,3 +71,30 @@ func CreatePlugin(app *application.Avalanche, subnetName string, pluginDir strin
 
 	return vmDestPath, binutils.CopyFile(vmSourcePath, vmDestPath)
 }
+
+// Downloads the target VM (if necessary) and copies it into the plugin directory
+func CreatePluginFromVersion(app *application.Avalanche, subnetName string, vm models.VMType, version string, vmid string, pluginDir string) (string, error) {
```

Comment:
probably split this line

Reply:
why not having similar interface than the previous one and get info from the sidecar (obtained from the subnetName). should expect both to use similar logic, and also, maybe, for one to call the one

Reply:
I cleaned up this line. We could try cleaning this up further later, but we probably don't want to do that in this PR.

Reply:
https://github.com/ava-labs/avalanche-cli/issues/325